packages/node_modules/@node-red/runtime/lib/nodes/flows/Subflow.js

```diff
@@ -22,6 +22,35 @@ const flowUtil = require("./util");
 
 var Log;
 
+
+/**
+ * Convert env var definition of subflow template and instance to env var dic
+ * @param {Hash} env1    env var definition of template
+ * @param {Hash} env2    env var definition of instance
+ * @return {Hash}        env var dic
+ */
+function env2Dic(env0, env1) {
```

Comment:
I had imagined `Subflow.getSetting` would check `this.subflowInstance.env` for a value (the instance-specific values) and if that was `undefined` check the value from the Subflow definition's env `this.subflowDef.env`, and if *that* was also `undefined` then defer to `this.parent.getSetting` as it does already (because it inherits `Flow.getSetting` currently). That removes the need to generate this dictionary of values. But I may be missing a benefit of doing this here.
 
If we do go this approach, the name needs changing - don't use `2` in place of `To`.